core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/repair/TestDefaultRepairConfigurationProvider.java

```diff
@@ -99,6 +99,12 @@ public Host getLocalHost()
             {
                 return localhost;
             }
+
+            @Override
```

Comment:
This is overriding a method with the exact same content.

Reply:
No,its needed since this has no relation to the thing in AbstractCassandraTest.